api/dev/library/OpenRA.lua

```diff
@@ -928,8 +931,22 @@ WVec = {
     Zero = nil;
 }
 
-
 ---@class actor
+--- Specifies whether the actor is in the world.
+---@field IsInWorld boolean
+--- The player that owns the actor.
+---@field Owner player
+--- Current actor stance. Returns nil if this actor doesn't support stances.
+---@field Stance string
+--- Current health of the actor.
+--- **Requires Trait:** [IHealth](https://docs.openra.net/en/release/traits/#ihealth)
```

Comment:
```suggestion
--- **Requires Trait:** [Health](https://docs.openra.net/en/release/traits/#health)
```

Reply:
The code does say `Requires<IHealthInfo>` though, which may or may not be the our `Health` trait 🤔  and as with the IMove case, there is no way to know which implementation this relies on (and it shouldn't rely on a specific implementation anyway).
I am not sure what to do here because `IHealth`/`IMove` may not be very understandable to non-programmers, but they *are* the correct things. I don't know how we could explain "you need one of the traits that implement this interface, which is: A, B, C, D"

Reply:
I just noticed that this is not a regression. We already have these interface links.